source/extensions/quic_listeners/quiche/platform/quic_flags_impl.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include <string>
+#include <vector>
+
+#include "extensions/quic_listeners/quiche/platform/flags_impl.h"
+
+#define GetQuicFlagImpl(flag) quiche::FLAGS_##flag->value()
+
+#define SetQuicFlagImpl(flag, value) quiche::FLAGS_##flag->SetValue(value)
+
+#define GetQuicReloadableFlagImpl(flag) quiche::FLAGS_quic_reloadable_flag_##flag->value()
+
+#define SetQuicReloadableFlagImpl(flag, value)                                                     \
+  quiche::FLAGS_quic_reloadable_flag_##flag->SetValue(value)
+
+#define GetQuicRestartFlagImpl(flag) quiche::FLAGS_quic_restart_flag_##flag->value()
+
+#define SetQuicRestartFlagImpl(flag, value) quiche::FLAGS_quic_restart_flag_##flag->SetValue(value)
+
+#define DEFINE_QUIC_COMMAND_LINE_FLAG_IMPL(type, flag, value, help)                                \
```

Comment:
TypedFlag is not connected to command line flags, can we add a comment to clarify?

Reply:
Done.